openmc/universe.py

```diff
@@ -823,12 +825,16 @@ def __init__(self,
                  universe_id=None,
                  name='',
                  auto_geom_ids=False,
-                 auto_mat_ids=False):
+                 auto_mat_ids=False, 
+                 mat_assignment={}):
         super().__init__(universe_id, name)
         # Initialize class attributes
         self.filename = filename
         self.auto_geom_ids = auto_geom_ids
         self.auto_mat_ids = auto_mat_ids
+        self.mat_assignment = mat_assignment
```

Comment:
Can we call this `material_overrides`? 

Also, since we'll need to add a doc string for this eventually, can you add that to `DAGMCUniverse` with a description of the data structure keys/values and it's purpose?